Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html

```diff
@@ -1,145 +1,114 @@
+    It follows that "Jen is sister of Mary" because the relation is symmetrically held between both instances. 
+
+Asymmetry provides that if (x) is related to (y) by object_property_3 then (y) CANNOT relate to (x) by object_property_3
+Ex: Asymmetric_Object_Property (Parent_of)
+    Object_Property_Assertion (parent_of, Vivian, Chris)  = "Vivian is a parent of Chris"
+    It holds that Chris CANNOT be a parent of Vivian. 
+
+The joint of these leads to unsatisfiability because an object property cannot relate mutually between instances while also only applying one direction between one to the other.
+Thus, the result will always be false because both cannot be achieved simulataneously. 
+
+
+4.) A-SYMMETRY + TRANSITIVITY = UNDECIDEABLE
+
+Asymmetry provides that if (x) is related to (y) by object_property_3 then (y) CANNOT relate to (x) by object_property_3
+Ex: Asymmetric_Object_Property (Parent_of)
+    Object_Property_Assertion (parent_of, Vivian, Chris)  = "Vivian is a parent of Chris"
+    It holds that Chris CANNOT be a parent of Vivian. 
+ 
+Transitive object-property goes according to the following:
+(x) relates to (y) by object-property 1 and (y) related to (z) by same object-property 1, 
+So (x) also relates to (z) by virture of object-property 1.
+x --> y, y --> z Therefore x --> z
+
+If there is an object property that holds as both asymmetrical and transitive, it is possible that an instance (y) could relate to (x) as both bearer and recipient
+of some object property if this object property is shared between other instances and (x). Thus the reasoner could not decide between whom it holds. 
+
+
+
+5.) TRANSITIVE + IRREFLEXIVE = UNDECIDEABLE
+
+Transitive object-property goes according to the following:
+(x) relates to (y) by object-property 1 and (y) related to (z) by same object-property 1, 
+So (x) also relates to (z) by virture of object-property 1.
+x --> y, y --> z Therefore x --> z
+
+Irreflexive hold that no individual can be connected by such an object property to itself.
+
+Syllogism: 
+
+Transitive Property Assertion: brother_of
+Irreflexive Property assertion: brother_of 
+
+    x is brother of y
+    y is brother of z
+    z is brother of x
+Therefore, x is brother to itself. 
+
+This leads to a contradiction because x cannot be brother to itself by virtue of irreflexivity. 
+
+
+
+6.) ASYMMETRICAL + REFLEXIVE = UNSATISFIABLE
+
+Asymmetry provides that if (x) is related to (y) by object_property_3 then (y) CANNOT relate to (x) by object_property_3
+Ex: Asymmetric_Object_Property (Parent_of)
+    Object_Property_Assertion (parent_of, Vivian, Chris)  = "Vivian is a parent of Chris"
+    It holds that Chris CANNOT be a parent of Vivian. 
+
+Reflexivity holds that an instance can relate to itself by some object property, (aRa) where a is some instance. 
+
+If (x) relates to some (y) and why, in turn, this (y) is able to point back to (x) by virtue of (x)'s reflexivity then this would lead to a contradiction. 
```

Comment:
Reflexivity implies that xRx for any instance x. Asymmetry, however, requires that if xRy, then yRx is prohibited. This creates a direct contradiction, as reflexivity mandates self-relations, which asymmetry forbids. Therefore, the combination is unsatisfiable.